README.md

```diff
@@ -51,6 +51,26 @@ The criteria for picking these event names included:
  - It uses names that are valid JavaScript identifiers, which allows using those as keys in JSON and avoids the need to quote keys in regular JS objects or function calls.
  - It doesn't overlap with any known existing events, so introducing these could be done in parallel to the existing API in each framework, optionally deprecating and eventually removing the previous API.
 
+### Event Data
+
+Based on the discussion in [#12](https://github.com/js-reporters/js-reporters/issues/12#issuecomment-120483356), there are two basic data structures:
+ - **Test**: A test holds basic information on a single test. It has the following set of required attributes
```

Comment:
Extend this a bit to cover BDD vocabulary. Something like: In BDD, this represents a single `it()`.